### ISSUE TITLE
Sonobus cannot detect system language fine (Linux)

### ISSUE THREAD

**Issue by Yajo** [sonosaurus/sonobus#92]
My system language is Spanish, and all other apps recognize it properly.

I just compiled 52cb09f and ran it locally:

![screenshot-2021-04-15-17-38-58](https://user-images.githubusercontent.com/973709/114906126-a5e7da00-9e11-11eb-9676-27b899164d71.png)

It appears in English, as you can see.

**Comment by essej** [comment]
Not sure why this is happening. As long as you can still manually change it to Spanish this won't be a high priority to fix...

**Comment by Yajo** [comment]
Yes, I understand it's not critical. It just would help new users for sure.
